extension/data_loader/file.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#ifndef _WIN32
+#include <unistd.h>
+#else
+#include <executorch/runtime/platform/compiler.h> // For ssize_t.
+#include <io.h>
+
+#include <windows.h>
+
+inline ssize_t pread(int __fd, void* __buf, size_t __nbytes, size_t __offset) {
```

Comment:
Please remove the leading underscores from the param names. Since this is our code and not part of the standard library, we shouldn't define any names beginning with underscore.

Reply:
Updated.